cornflow-server/cornflow/endpoints/execution.py

```
@@ -488,23 +639,27 @@ def delete(self, idx):
     @authenticate(auth_class=Auth())
     @Auth.dag_permission_required
     def post(self, idx):
+        if self.orch_type != AIRFLOW_BACKEND:
+            return {
+                "message": f"This feature is not available for {self.orch_const['name']}"
+            }, 200
```

This should probably raise an exception or return an error status code